src/ui/lib/Button.tsx

```diff
@@ -27,7 +27,7 @@ export function Button({
 }: ClickableProps): JSX.Element {
   return (
     <button
-      className={`flex w-9 w-auto min-w-min appearance-none items-center justify-center justify-center rounded-full bg-primary py-2 px-4 text-white hover:scale-[1.01] ${
+      className={`flex min-w-min appearance-none items-center justify-center rounded-full bg-[#321D47] py-2 px-4 text-sm text-white  hover:scale-[1.01] ${
```

Comment:
I'll pass this as className when the dark buttons are intended (delegate Cards?!) since this is a custom color and the component was intended to be generic. As I'm checking the polkadot.network seems the buttons on the header are primary color and there some buttons that jave dark theme, but I believe the default should be primary color.

Reply:
AFAIK the settlement around the 'magenta' color with Telekom says we can't use it on 'surfaces'. Which means we can use it in fonts, lines, etc, but not on buttons. I'll double check with brand people - especially b/c staking uses pink all over the place, however, that's a different pink than what you have as primary (which is the correct one I think)

Reply:
you're right, this settlement/limitation only applies to the website, not to the apps. 
i changed the button back to pink, added a secondary color and a secondary button version - i also updated the [designs in figma](https://www.figma.com/file/ke1WLYloUbG7XyO1N6OVBp/Open-Gov-MVP?node-id=1%3A1466&t=RnAD4ES6dEBHAoIS-11) to not have pink all over the place, to create some hierarchy